live-examples/html-examples/forms/meta.json

```diff
@@ -1,5 +1,13 @@
 {
     "pages": {
+        "button": {
+            "baseTmpl": "tmpl/live-tabbed-tmpl.html",
+            "exampleCode": "live-examples/html-examples/forms/button.html",
+            "cssExampleSrc": "live-examples/html-examples/forms/css/button.css",
+            "fileName": "button.html",
+            "title": "HTML Demo: &lt;button&gt;",
```

Comment:
FYI: Once PR https://github.com/mdn/interactive-examples/pull/777 is merged, you no longer need to manually escape the `<` and `>`